dataplicity/m2m/commandservice.py

```diff
@@ -38,54 +38,83 @@ def __init__(self, channel, command):
     def __repr__(self):
         return self._repr
 
+    @classmethod
+    def send_error(cls, channel, status, msg, **extra):
```

Comment:
> codacity would complain

Interesting... I don't think we should be doing strange things to the code to make codacy happy. How did it complain? I suppose it said something about it not using `self`? In that case, perhaps it would make more sense to make this a function instead. That way, the duplcation between the two modules here might be easier to resolve too.

> I'm reserving that as an extension mechanism.

Ah!